app_base.htm

```diff
@@ -0,0 +1,79 @@
+<%inherit file="homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+##set the active tab dynamically with code
```

Comment:
hey nice comment to tell us what is going on here

Reply:
Thank you! I always try to add comments when I remember to do so.